src/fdc3/Directory.Build.props

```diff
@@ -0,0 +1,3 @@
+<Project>
+	<Import Project="../Directory.Build.props" Condition="Exists('../Directory.Build.props')" />
```

Comment:
Don't remove now that they are there, but these files (with the `Import` for the parent directory) are only necessary when they have content, otherwise the `Directory.Build.props` file closest to the project is imported automatically.

Reply:
Understood. The reason why I have added to this folder is because the DesktopAgent project had the Directory.Build.props file before, and it was searching for the props file above the folder, and that's why It couldn't resolve the common properties when I added the targets for the project.